src/js/entities-service/graphql.js

```diff
@@ -0,0 +1,40 @@
+import BaseEntity from 'js/base/entity-service';
+import fetcher from 'js/base/fetch';
+
+const Entity = BaseEntity.extend({
+  radioRequests: {
+    'fetch:icd:byTerm': 'fetchIcdByTerm',
+  },
+  fetchIcdByTerm(term) {
+    const variables = { term };
+    const query = `query ($term: String!) {
+      icdCodes(term: $term) {
+        code
+        description
+        hcc_v24
+        hcc_v28
+        parent {
+          code
+          description
+        }
+        children {
+          code
+          description
+        }
```

Comment:
@zfixler do you need the parent/child in the form?

Reply:
I think it's very likely that we will need it for [this story.](https://app.shortcut.com/roundingwell/story/48868/ability-to-flag-invalid-codes)

It's very likely we will want to show the children as options in the form if a parent is added.